client/send-video.cc

```diff
@@ -51,6 +60,7 @@ static int usage(const char *progname) {
             "\t-g <width>x<height>[+<off_x>+<off_y>[+<layer>]] : Output geometry. Default 20x20+0+0\n"
             "\t-h <host>          : Flaschen-Taschen display hostname.\n"
             "\t-l <layer>         : Layer 0..15. Default 0 (note if also given in -g, then last counts)\n"
+            "\t-r<repeat-secs>    : Repeat for at least n seconds\n"
```

Comment:
similar to the roughly similar function in send-image, I'd call this option `-t` as it more reflects that this is a time we're talking about here.

`-r` would also be useful as a count. So `-r 5` means to play the video five times.

Also make sure to leave a space after the option and the `<..>` value, similar to the one above, where we have a space between `-l` and `<layer>`.